package.json

```diff
@@ -62,7 +62,8 @@
     "makelogs": "makelogs",
     "mocha": "mocha",
     "mocha:debug": "mocha --debug-brk",
-    "sterilize": "grunt sterilize"
+    "sterilize": "grunt sterilize",
+    "uiFramework:start": "./node_modules/.bin/webpack-dev-server --config src/ui_framework/webpack.config.js --hot --inline --content-base src/ui_framework/public/"
```

Comment:
You don't need `./node_modules/.bin/`when running npm scripts, the `node_modules/.bin` is included in the path by default in that case.